From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jake Weinstein <jake@aospa.co>
Date: Tue, 13 Sep 2022 05:14:08 +0900
Subject: [PATCH] arm64: configs: moto-holi: Disable KALLSYMS_ALL

We do not need to load all symbols into the kernel image.
This takes a significant amount of compile time.

Signed-off-by: chandu078 <chandudyavanapelli03@gmail.com>
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 arch/arm64/configs/vendor/lineage_moto-holi.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/vendor/lineage_moto-holi.config b/arch/arm64/configs/vendor/lineage_moto-holi.config
index 7005c4f587a6..5cc4db6202e1 100644
--- a/arch/arm64/configs/vendor/lineage_moto-holi.config
+++ b/arch/arm64/configs/vendor/lineage_moto-holi.config
@@ -80,3 +80,4 @@ CONFIG_NR_CPUS=8
 # CONFIG_CPU_FREQ_GOV_CONSERVATIVE is not set
 # CONFIG_UNMAP_KERNEL_AT_EL0 is not set
 # CONFIG_HARDEN_BRANCH_PREDICTOR is not set
+# CONFIG_KALLSYMS_ALL is not set
-- 
2.39.5

